src/main/java/com/three_tech_solutions/slot_app/data/models/Slot.java

```diff
@@ -24,6 +24,13 @@ public class Slot {
     private User user;
     @OneToMany(cascade = CascadeType.ALL)
     private List<SpecificSlot> specificSlots;
+    @ManyToMany
+    @JoinTable(
+            name = "slot_student",
+            joinColumns = @JoinColumn(name = "slot_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_id")
+    )
+    private List<Student> students;
```

Comment:
Esto en vez de una lista, podria ser un `Set<Student>` para evitar estudiantes duplicados en un mismo turno

```suggestion
    private Set<Student> students;
```